drivers/i2c/i2c_ll_stm32.h

```diff
@@ -24,6 +24,7 @@ struct i2c_stm32_data {
 #ifdef CONFIG_I2C_STM32_INTERRUPT
 	struct k_sem device_sync_sem;
 #endif
+	struct k_sem		sem_lock;
```

Comment:
Please fix spacing between k_sem and sem_lock

Reply:
ok